deploy/config/microsoft/cloudconfig.yml

```diff
@@ -25,11 +24,12 @@ cloud:
             maxCount: 10
 
     managementPrincipals:
-      - id: 2e092785-472a-4ea1-9700-4d15646d9e91
+      - objectId: 2e092785-472a-4ea1-9700-4d15646d9e91
         kind: ServicePrincipal
-      - id: c0e60aba-35f0-4778-bc9b-fc5d2af14687
+      - objectId: 36043a62-1383-4012-95aa-44da0a4d8012
```

Comment:
I think it will be ok. I don't see us using this for email distribution. And if there is an issue with the group, one can always email the owners directly.